gnovm/tests/files/access1_stdlibs.gno

```diff
@@ -9,4 +9,4 @@ func main() {
 }
 
 // Error:
-// main/files/access1.gno:8: cannot access gno.land/p/demo/testutils.testVar2 from main
+// main/files/access1_stdlibs.gno:8: cannot access gno.land/p/demo/testutils.testVar2 from main
```

Comment:
This is a temporary solution because some tests fail in the type checker due to two other issues. see: (https://github.com/gnolang/gno/issues/2268) (https://github.com/gnolang/gno/issues/2267)

Reply:
need another issue to track and to rename these, so it isn't temporary.